sspl_test/mock_data/disk-groups.txt

```diff
@@ -1,93 +1,104 @@
-        "create-date":"1980-09-22 02:05:47",
-        "create-date-numeric":338436347,
-        "cache-read-ahead":"0B",
-        "cache-read-ahead-numeric":0,
-        "cache-flush-period":0,
-        "read-ahead-enabled":"Enabled",
-        "read-ahead-enabled-numeric":1,
-        "write-back-enabled":"Enabled",
-        "write-back-enabled-numeric":1,
-        "job-running":"",
-        "current-job":"",
-        "current-job-numeric":0,
-        "current-job-completion":"",
-        "num-array-partitions":4,
-        "largest-free-partition-space":"0B",
-        "largest-free-partition-space-numeric":0,
-        "num-drives-per-low-level-array":1,
-        "num-expansion-partitions":0,
-        "num-partition-segments":0,
-        "new-partition-lba":"0B",
-        "new-partition-lba-numeric":0,
-        "array-drive-type":"SAS MDL",
-        "array-drive-type-numeric":11,
-        "disk-description":"SAS MDL",
-        "disk-description-numeric":11,
-        "is-job-auto-abortable":"false",
-        "is-job-auto-abortable-numeric":0,
-        "serial-number":"00c0ff27cb0e0000fb202c1400000000",
-        "blocks":93672488960,
-        "disk-dsd-enable-vdisk":"Disabled",
-        "disk-dsd-enable-vdisk-numeric":0,
-        "disk-dsd-delay-vdisk":0,
-        "scrub-duration-goal":0,
-        "adapt-target-spare-capacity":"0B",
-        "adapt-target-spare-capacity-numeric":0,
-        "adapt-actual-spare-capacity":"0B",
-        "adapt-actual-spare-capacity-numeric":0,
-        "adapt-critical-capacity":"0B",
-        "adapt-critical-capacity-numeric":0,
-        "adapt-degraded-capacity":"0B",
-        "adapt-degraded-capacity-numeric":0,
-        "adapt-linear-volume-boundary":0,
-        "pool-sector-format":"512e",
-        "pool-sector-format-numeric":1,
-        "health":"Degraded",
-        "health-numeric":1,
-        "health-reason":"The disk group is not fault tolerant. Reconstruction cannot start because there is no spare disk available of the proper type and size.",
-        "health-recommendation":"- For non-ADAPT disk groups, replace the disk with one of the same type (SSD, enterprise SAS, or midline SAS) and the same or greater capacity. For ADAPT disk groups, replace the disk with one of the same type. For continued optimum I/O performance, the replacement disk should have performance that is the same as or better than the one it is replacing.  - For non-ADAPT disk groups, configure the new disk as a spare so the system can start reconstructing the disk group.  - For ADAPT disk groups, expand the disk group to include the new disk as spare space.  - To prevent this problem in the future, configure one or more additional disks as spare disks in non-ADAPT disk groups, or increase the spare space size of ADAPT disk groups.  - If the disk group is being expanded, reconstruction will start after the expansion is complete."
-      }
-    ]
-  },
-    "status_code": 200
+        "api-response": {

```

Comment:
What is the change here? If you are reformatting the file, it would be much better to read if formatted two space indentation.

Reply:
Status keys is missing in current mocked data.
Line 90 to 102 having it.
Sure, I will intend with 2 spaces.

Reply:
Addressed.
Done.